### PR TITLE
Automate bsscript version

### PR DESCRIPTION
**Pull request by mseidl**
## What does this PR change
This get's the bootstrap script  versions for supportconfig and dynamically adds the version to the bootstrap script header.
#16805 

- [ ] **DONE**

## Documentation
- No documentation needed: 

- [ ] **DONE**

## Test coverage
- N
## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: 

- [ ] 
## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: 

- [ ] **DONE**

## Test coverage
- No tests: 

- [ ] **DONE**

## Links 

https://bugzilla.suse.com/show_bug.cgi?id=11863361

### CODE REVIEW

**Review by mcalmer — APPROVED** [review]

File: spacewalk/certs-tools/rhn_bootstrap_strings.py [hunk 1]
```diff
@@ -252,8 +253,13 @@ def getHeader(productName, options, orgCACert, isRpmYN, pubname, apachePubDirect
     # Removing paths from options.gpg_key
     org_gpg_key = ",".join([os.path.basename(gpg_key) for gpg_key in options.gpg_key.split(",")])
     exit_call = " " if options.activation_keys or options.salt else "exit 1"
+    with cfg_component('web') as CFG:
+        version = CFG.version
+        if isUyuni:
+            version = CFG.uyuni
```
Comment (meaksh) [inline spacewalk/certs-tools/rhn_bootstrap_strings.py:259]: This logic is wrong here as you are not calling `isUyuni()` but just checking it exists. 

I think what you need is:

```suggestion
        version = CFG.uyuni if isUyuni() else CFG.version
```

**Review by meaksh — CHANGES_REQUESTED** [review]
Thanks for this. I just found a minor problem with the logic

File: spacewalk/certs-tools/rhn_bootstrap_strings.py (hunk 1 above)
Comment (agraul) [reply spacewalk/certs-tools/rhn_bootstrap_strings.py:259]: The "ugly" bit here (unrelated to you suggestion, that's toally valid!) is that `CFG.uyuni` is defined as `CFG.version.uyuni` in `rhn_web.conf` but accessed as `CFG.uyuni`.

**Review by meaksh — APPROVED** [review]

**Review by agraul — APPROVED** [review]
It seems that I'm the only one who dislikes accessing `web.version.uyuni` with `CFG.uyuni`, so let's keep it like that!

**Pull request merged**